apps/test/npm-debug.log

```diff
@@ -0,0 +1,28 @@
+0 info it worked if it ends with ok
```

Comment:
You probably didn't mean to check in npm-debug.log, right?  We might need to put this in our .gitignore